### PR TITLE
Remove notion of "private" selector from the Include/Exclude filter.

### PR DESCRIPTION
**Pull request by kwilczynski**
This commit removes any mention of the "private" selector from the Include and
Exclude filters, as it is not supported by the upstream `go-sockaddr` project.

Signed-off-by: Krzysztof Wilczynski <kw@linux.com>

### CODE REVIEW

**Comment by kwilczynski** [comment]
Fixes #328.

File: docs/content/functions/sockaddr.md [hunk 1]
```diff
@@ -372,4 +370,4 @@ $ gomplate -i '{{ sockaddr.GetInterfaceIPs "en0" }}'
 10.0.0.28 fe80::1f9a:5582:4b41:bd18
 ```
 
-[RFC 6890]: http://tools.ietf.org/html/rfc6890
+[1]: http://tools.ietf.org/html/rfc6890
```
Comment (hairyhenderson) [inline docs/content/functions/sockaddr.md]: why this change?

Comment (hairyhenderson) [reply docs/content/functions/sockaddr.md]: The `[RFC 6890][]` links are totally valid markdown - they all produce correct HTML:

```html
<a href="http://tools.ietf.org/html/rfc6890">RFC 6890</a>
```

There's an `[RFC 1918][]` link, on the other hand, which is invalid, because there's no associated `[RFC 1918]: http://tools.ietf.org/html/rfc1918` footer (which is a separate bug).

Since this PR is about removing `private`, please revert the `RFC 6890` changes.

Comment (kwilczynski) [reply docs/content/functions/sockaddr.md]: Ah, that's my bad. I meant the broken one. Apologies.

**Comment by kwilczynski** [comment]
Hi @hairyhenderson,

TL;DR
I removed the wrong fix to the markdown label (will send other Pull Request to resolve that) and only pushed change to remove the mention of "private". 

There are some lessons learned here for me :)

1) The economy class on AirFrance does not have enough space for 15" MacBook Pro
2) The "tap to click" and turbulences do not complete each-other well :) Doh.
3) Paying $10 (or so) for 200 MB of data was silly (albeit, I can't sleep doing the flights so talked myself into trying to be more "productive", so to speak)

Re: closing the Pull Request - for some cosmic reason I managed to tap the trackpad by when plane was flying through a small area of turbulence when the cursor was right above "Close pull request" when I was about to write a response. It kind of made me laugh a bit back then :) Apologies for troubles.
